modin/pandas/accessor.py

```diff
@@ -108,8 +110,9 @@ def density(self):
 
     @classmethod
     def from_spmatrix(cls, data, index=None, columns=None):
-        return cls._default_to_pandas(
-            pandas.DataFrame.sparse.from_spmatrix, data, index=index, columns=columns
+        ErrorMessage.default_to_pandas("`from_spmatrix`")
```

Comment:
@YarShev which accessor do you mean?

Reply:
@mvashishtha, I meant `SparseFrameAccessor` but since all the other accessors have default to pandas implementations for their methods/properties we should add the warning to those too.

Reply:
Those accessors already warn that they are defaulting to pandas. They use the dataframe's or series's [default to pandas method](https://github.com/modin-project/modin/blob/7e85c5df016f8aad20c7f8bea64910fae8793775/modin/pandas/base.py#L431), which warns about the default to pandas.